salt/kafka-tool/init.sls

```diff
@@ -0,0 +1,73 @@
+#read all pillar data
+
+{% set install_dir = pillar['pnda']['homedir'] %}
+{% set packages_server = pillar['packages_server']['base_uri'] %}
+{% set release_directory = salt['pillar.get']('kafkatool:release_dir', '/opt/pnda') %}
+{% set release_version = salt['pillar.get']('kafkatool:version', '0.3.0')  %}
```

Comment:
should be kafkatool:release_version with default value to v0.2.0

Reply:
Modified